DeviceBridgeE2E/src/device-bridge.spec.ts

```diff
@@ -0,0 +1,205 @@
+
+    // Ensure get works
+    var response = await t.context.ctx.deviceBridgAPI.getConnectionStatusSubscription(t, t.context.device.id);
+    assert(callbackUrl, response.callbackUrl)
+
+    // Ensure connection created event when a sub created
+    await t.context.ctx.deviceBridgAPI.createCMDSubscription(t, t.context.device.id, callbackUrl);
+    await sleep(3000);
+    var invocationValue = await t.context.ctx.deviceBridgAPI.getEcho(t, t.context.device.id);
+    var invocationValueBody = JSON.parse(invocationValue.body);
+    t.is(invocationValueBody.status, "Connected");
+    t.is(invocationValueBody.eventType, "ConnectionStatusChange");
+    t.is(invocationValueBody.deviceId, t.context.device.id);
+
+    // Ensure connection deleted event when a sub deleted
+    await t.context.ctx.deviceBridgAPI.deleteCMDSubscription(t, t.context.device.id);
+    await sleep(2000);
+    var invocationValue = await t.context.ctx.deviceBridgAPI.getEcho(t, t.context.device.id);
+    var invocationValueBody = JSON.parse(invocationValue.body);
+    t.is(invocationValueBody.status, "Disabled");
+    t.is(invocationValueBody.eventType, "ConnectionStatusChange");
+    t.is(invocationValueBody.deviceId, t.context.device.id);
+});
+
+test.serial('Test device desired property update callback', async t => {
+    // Create subscription to Azure Function
+    const callbackUrl = `${t.context.ctx.callbackUrl}&deviceId=${t.context.device.id}`;
+    await t.context.ctx.deviceBridgAPI.createDesiredPropertySubscription(t, t.context.device.id, callbackUrl)
+    await sleep(3000);
+
+    // Ensure get works
+    var response = await t.context.ctx.deviceBridgAPI.getDesiredPropertySubscription(t, t.context.device.id);
+    t.is(response.body.callbackUrl, callbackUrl)
+    t.is(response.body.status, "Running");
+    var propTestValue = "test";
+    await t.context.ctx.publicAPI.setProperties(t, t.context.device.id, {rwProp: propTestValue});
+    await sleep(3000);
+    var invocationValue = await t.context.ctx.deviceBridgAPI.getEcho(t, t.context.device.id);
+    var invocationValueBody = JSON.parse(invocationValue.body);
+    t.is(invocationValueBody.desiredProperties.rwProp, propTestValue);
+    t.is(invocationValueBody.eventType, "DesiredPropertyUpdate");
+    t.is(invocationValueBody.deviceId, t.context.device.id);
+
+    // Ensure connection deleted event when a sub deleted
+    await t.context.ctx.deviceBridgAPI.deleteDesiredPropertySubscription(t, t.context.device.id);
+    await sleep(2000);
+    var invocationValue = await t.context.ctx.deviceBridgAPI.getEcho(t, t.context.device.id);
+    var invocationValueBody = JSON.parse(invocationValue.body);
+    t.is(invocationValueBody.status, "Disabled");
+    t.is(invocationValueBody.eventType, "ConnectionStatusChange");
+    t.is(invocationValueBody.deviceId, t.context.device.id);
+});
+
+test.serial('Test device to cloud messaging', async t => {
+    const temperatureValue = 120;
```

Comment:
Can you use a random value (to make sure we're not picking up the value of a previous test execution)